src/env/env.rs

```diff
@@ -179,6 +214,21 @@ pub fn uumain(args: Vec<String>) -> i32 {
         }
     }
 
+    for file in &opts.files {
+        let conf = match Ini::load_from_file(file) {
```

Comment:
I'm fine with adding this feature (another interesting one might be to enable processing env vars given through `stdin`; it would need to be behind an option flag to prevent breaking things though), but I'm uncertain whether we should use INI files for the input format.

I think it would make more sense for it to be just space-separated/newline-separated values (and maybe comments).  INI files have sections, which don't really make sense in this context, so I think would be weird to accept them as input.

Additionally I think the files array should check for `-`, as that represents `stdin` in other utilities.

Reply:
If the sections are ignored, an INI file is essentially just a basic *nix config file, which is why I chose the `rust-ini` crate. Using `rust-ini` just gives both INI and basic config file compatibility for "free"; user can choose.

Here, when reading the file, all sections, if they exist, are ignored, essentially being treated as comment lines.

If you have another suggestion for a reader, I'd be glad to explore it.

I do think this could easily be extended to use `-` as `stdin`. But, I'm not sure what you mean about "behind an option flag". I would just extend the `--file` option to read `--file -` as "read from `stdin`"... I'll go ahead an try to implement that now.

Reply:
You can ignore the comment about a new flag for reading from `stdin`.  I thought of that before realizing that you could just do `--file -`.

Specifically, I'd prefer something like `.env` file parsing, but the only crate I know about that can parse them (`dotenv`) doesn't seem to expose a parser (well, it sort of does, but it just loads stuff into the environment without letting you look at the values).

If you can't find anything to handle `.env` files (that looks good), we can just keep the INI file parsing for now.  I'd put a comment in saying that support for parsing sections may be removed in the future though.

Reply:
Here's a working update.
I'll look through some crates to see if I an find something that parses "plain config" .env-type files.

Reply:
This looks pretty good.  I'd be willing to merge it now if you can't find a good alternative parser.

Reply:
Yeah, I spent some time perusing crates last night and don't see anything applicable. I'll change the help text to emphasize that the file is a ".env"-style configuration file, and I'll add a comment that the "INI" compatibility is coincidental, not intended. Hopefully then, it could be changed out if some better crate pops up in the future.

Reply:
I'll squash and rebase to current master a little later today.